src/components/FooterComponent/Footer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import style from './footer.scss';
+
+const Footer = () => (
+  <footer>
+    <div className={style.container}>
+      <div className={style.information}>
+        <div>
+          <h4>Follow Us</h4>
+          <p>
+            <img src="https://res.cloudinary.com/allebd/image/upload/v1567001062/dahlia/icons8-facebook-f-50.png" alt="facebook logo" />
```

Comment:
Why aren't you using fontawesome icon here

Reply:
Initially wanted to make use of the same reference on Figma, but would update it to font-awesome